opencti-platform/opencti-front/src/private/components/common/containers/ContainerAddStixCoreObjects.jsx

```diff
@@ -94,18 +96,71 @@ const ContainerAddStixCoreObjects = (props) => {
     ],
   };
 
-  const [filters, setFilters] = useState(
-    targetStixCoreObjectTypes
-    && !(targetStixCoreObjectTypes.includes('Stix-Domain-Object') || targetStixCoreObjectTypes.includes('Stix-Cyber-Observable'))
-      ? targetEntityTypesFilterGroup
-      : emptyFilterGroup,
+  const isTypeDomainObject = (types) => {
+    return !types
```

Comment:
So it's `true` if we don't give any type? That feels strange

Reply:
yes if no types, there is no 'inital filter by default' (if it was the question)